README.md

```diff
@@ -1,42 +1,42 @@
 git-dit - the distributed issue tracker for git
 
-This repository contains the "git-dit" project, a distributed issue tracking
-system using git. For a system overview and conceptual information, refer to the
-[documentation](doc/README.md). For a more practical documentation, refer to the
-[man page](git-dit.1.md).
+[![Build Status](https://travis-ci.org/neithernut/git-dit.svg?branch=master)](https://travis-ci.org/neithernut/git-dit)
+[![Issue Stats](https://img.shields.io/badge/issues%20closed%20in-7%20days-yellowgreen.svg?style=flat-square)](http://www.issuestats.com/github/neithernut/git-dit)
+[![](https://tokei.rs/b1/github/neithernut/git-dit)](https://github.com/neithernut/git-dit)
+[![license](https://img.shields.io/github/license/neithernut/git-dit.svg?maxAge=2592000?style=flat-square)]()
```

Comment:
I don't know whether I told you before, but I'm in strong opposition to such badges. For example:
 * They ate specific to github.
 * The README may not only be read on github but also offline. In such a situation, the badges are rather annoying.
 * The state the badges are displaying is the current state, not the state of the commit which may be viewed.